Wd 21767

petesfrench opened the pull request:
## Done

- Removes tabotronic.js and updates any pages using it to use tabbed-content.js
- Removes tabbed-contents.js from main.js and include it only where it is needed

## QA

- Test all pages that use tabs, these are:

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

britneywwc commented:
- [ ] [/download/server/multipass](https://ubuntu-com-15055.demos.haus/download/server/multipass) links to [/download/server#instant-vms](https://ubuntu-com-15055.demos.haus/download/server#instant-vms) but it doesn't open the `Instant VMs` page
- [ ] [/resources](https://ubuntu-com-15055.demos.haus) redirects to /blog. There are no tabs to QA there
- [ ] Left a comment [here](https://docs.google.com/spreadsheets/d/15vMEEydIDrm_1qz5t2KLLSQeK62GNHI9BSxZlz2mgpk/edit?disco=AAABhGZIycU) for /containers page
- [ ] These pages are throwing 500
  - [/security/security-standards](https://ubuntu-com-15055.demos.haus/security/security-standards)
  - [/certified/platforms/platform-details](https://ubuntu-com-15055.demos.haus/certified/platforms/platform-details)
  - [/certified/model-details](https://ubuntu-com-15055.demos.haus/certified/model-details)
- [ ] [/credentials/syllabus](https://ubuntu-com-15055.demos.haus/credentials/syllabus) is an incomplete page, no tabs to QA
- [ ] [/appliance/adguard](https://ubuntu-com-15055.demos.haus/credentials/syllabus) does not have tabs
- [ ] Tab hashes are not attached to the URL when selected e.g on [/appliance/adguard/intel-nuc](https://ubuntu-com-15055.demos.haus/appliance/adguard/intel-nuc), tab hashes are not attached to the URL when tabs are selected
  - [ ] Same for all appliance pages

petesfrench commented:
> * [ ]  [/download/server/multipass](https://ubuntu-com-15055.demos.haus/download/server/multipass) links to [/download/server#instant-vms](https://ubuntu-com-15055.demos.haus/download/server#instant-vms) but it doesn't open the `Instant VMs` page
I also renamed the partials used here to have leading underscores ex. _multipass.html
> * [ ]  [/resources](https://ubuntu-com-15055.demos.haus) redirects to /blog. There are no tabs to QA there
I have removed this page
> * [ ]  These pages are throwing 500
>   
>   * [/security/security-standards](https://ubuntu-com-15055.demos.haus/security/security-standards)
This one works for me
>   * [/certified/platforms/platform-details](https://ubuntu-com-15055.demos.haus/certified/platforms/platform-details)
>   * [/certified/model-details](https://ubuntu-com-15055.demos.haus/certified/model-details)
These two are in fact not pages, my mistake. My grep I used to find the pages was a little greedy.
> * [ ]  [/credentials/syllabus](https://ubuntu-com-15055.demos.haus/credentials/syllabus) is an incomplete page, no tabs to QA
Can ignore this page also
> * [ ]  [/appliance/adguard](https://ubuntu-com-15055.demos.haus/appliance/adguard) does not have tabs
Can ignore
> * [ ]  Tab hashes are not attached to the URL when selected e.g on [/appliance/adguard/intel-nuc](https://ubuntu-com-15055.demos.haus/appliance/adguard/intel-nuc)
Should be fixed now

britneywwc reviewed (approved):
Thanks for the changes, looks good! 
https://ubuntu-com-15055.demos.haus/appliance/lxd/vm is still throwing 500 for me, might be a server issue
